docs/_scripts/base.sh

```diff
@@ -1,20 +1,16 @@
-#!/usr/bin/env bash
 echo "Setting environment variables for default hostname/port for the API and the Content app"
 export BASE_ADDR=${BASE_ADDR:-http://localhost:24817}
-export CONTENT_ADDR=${CONTENT_ADDR:-http://localhost:24816}
```

Comment:
I think we need this back

Reply:
actually we do not use anywhere content_addr